src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -133,12 +133,13 @@ public EventStreamController(final EventTypeRepository eventTypeRepository,
         List<Cursor> cursors = null;
         if (cursorsStr != null) {
             try {
-                cursors = jsonMapper.readValue(cursorsStr, new TypeReference<ArrayList<Cursor>>() {});
```

Comment:
Was there anything changed in behavior of `getStreamingStart`? Or are there only improvements and refactoring? Because I could not identify any fix/change in this method.

Reply:
@v-stepanov There was small refactoring, that didn't change anything. It was made because it was hard to analyse what is going on, therefore I had to separate cursor conversion and topic repository requests.